browser/package.json

```diff
@@ -8,7 +8,7 @@
     "@types/jquery.contextmenu": "^1.7.38",
     "@types/jqueryui": "^1.12.21",
     "@types/mocha": "8.2.0",
-    "@types/node": "14.14.25",
+    "@types/node": "18.11.09",
     "@types/offscreencanvas": "^2019.7.3",
```

Comment:
Please do not change that if possibleour CI as we experienced in the past. Why is that needed?

Reply:
I upgraded version for structuredClone() function looks nice for cloning arrays, maps.

Reply:
ah it's just types :) it'as fine then